go/kbfs/idutil/daemon_local.go

```diff
@@ -565,12 +565,13 @@ func (dl *DaemonLocal) RemoveAssertionForTest(assertion string) {
 // AddTeamWriterForTest adds a writer to a team.  Should only be used
 // by tests.
 func (dl *DaemonLocal) AddTeamWriterForTest(
-	tid keybase1.TeamID, uid keybase1.UID) (kbname.NormalizedUsername, error) {
+	tid keybase1.TeamID, uid keybase1.UID) (
+	kbname.NormalizedUsername, bool, error) {
```

Comment:
There's a `bool` so maybe should name these? `normalizedUsername kbname.NormalizedUsername, isImplicit bool, err error`